middleware/app.js

```diff
@@ -26,8 +29,45 @@ app.delete('/', function(_, res) {
   res.send(`DELETE request homepage - localhost:${PORT}`);
 });
 
+// List all vehicles from blockchain
+app.get('/vehicles', function(req, res) {
+  (async () => {
+    const privateKey = req.headers.authorization;
+    const VehicleClient = require('./lib/vehicle-client.js');
+    const client = new VehicleClient(
+      privateKey,
+    );
+  
+    const resVehicleList = await client.listVehicles();
+    console.log(resVehicleList);
+    res.send(resVehicleList);
+  })();
+});
+
+// Post a new vehicle to the blockchain
+app.post('/vehicles', function(req, res) {
+  (async () => {
+    const privateKey = req.headers.authorization;
+    const VehicleClient = require('./lib/vehicle-client.js');
+    const client = new VehicleClient(
+      privateKey,
+    );
+  
+    const newVehicle = {
+      vin: req.body.vin,
+      model: req.body.model,
+      color: req.body.color,
+      status: req.body.status,
+    };
+  
+    const resPostVehicle = await client.createVehicle(newVehicle);
+    console.log(resPostVehicle);
+    res.send(resPostVehicle);
+  })();
+});
+
 app.use('/keys', require('./routes/keys'));
 
 app.listen(PORT, () =>
   console.log(`Server listening on port ${IS_PRODUCTION ? PORT : '8080'}.`),
-);
+);
```

Comment:
@ej3076 yelled at me for not having white space at the end of my file, so now I have to do it to you too lol